src/data/List.js

```diff
@@ -467,6 +467,21 @@ listProto.getProvider = function () {
     return this._rawData;
 };
 
+// Get store data
+listProto.getStorage = function () {
+    return this._storage;
+};
+
+/**
+ *  Get specific item's data
+ *
+ * @param {string} dim dim name
+ * @return {Array} item data
+ */
+listProto.getStorageItem = function (dim) {
+    return [].slice.call(this._storage[dim][0]);
```

Comment:
`this._storage[dim]` will be separated into sevaral chunks when data is dynamically appended with `appendData` method.
 
Should not hard code the `chunkIndex` to `0` here.